<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QFrame" name="btn_frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame, QLabel, QToolTip { border: 2px solid black; border-radius: 3px; padding: 1px; background-color: rgb(0, 61, 90); color: white; }
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QPushButton" name="StartButton">
            <property name="minimumSize">
             <size>
              <width>56</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">	QPushButton {
	background-color: #138086;
    border-style: outset;
    border-width: 1px;
    border-radius: 4px;
    border-color: black;
    font: bold 12px;
    min-width: 3em;
    padding: 3px;
	color:rgb(255,255,255);
}
QPushButton:pressed {
	background-color:rgb(100,170,255)
}</string>
            </property>
            <property name="text">
             <string>Strat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="takesampleButton">
            <property name="minimumSize">
             <size>
              <width>56</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">	QPushButton {
	background-color: #138086;
    border-style: outset;
    border-width: 1px;
    border-radius: 4px;
    border-color: black;
    font: bold 12px;
    min-width: 3em;
    padding: 3px;
	color:rgb(255,255,255);
}
QPushButton:pressed {
	background-color:rgb(100,170,255)
}</string>
            </property>
            <property name="text">
             <string>Take sample</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PredictButton">
            <property name="minimumSize">
             <size>
              <width>56</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">	QPushButton {
	background-color: #138086;
    border-style: outset;
    border-width: 1px;
    border-radius: 4px;
    border-color: black;
    font: bold 12px;
    min-width: 3em;
    padding: 3px;
	color:rgb(255,255,255);
}
QPushButton:pressed {
	background-color:rgb(100,170,255)
}</string>
            </property>
            <property name="text">
             <string>Get Predictions</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphHomeqgraphic"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionCalibrate_Spectrometer">
   <property name="text">
    <string>Calibrate Spectrometer</string>
   </property>
  </action>
  <action name="actionStart_Spectroscopy">
   <property name="text">
    <string>Start Spectroscopy</string>
   </property>
  </action>
  <action name="actionTake_sample_for_ML">
   <property name="text">
    <string>Take sample for ML</string>
   </property>
  </action>
  <action name="actionGet_Prediction">
   <property name="text">
    <string>Get Prediction</string>
   </property>
  </action>
  <action name="actionPick_a_Model">
   <property name="text">
    <string>Pick a Model</string>
   </property>
  </action>
  <action name="actionquit">
   <property name="text">
    <string>Quit Application</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
